### DISCUSSION TITLE
Force/manual session save?

### DISCUSSION THREAD

**Discussion by boomboxnation** [crazy-max/docker-rtorrent-rutorrent#382] [General]
Does anyone know of a way to do a manual whatever RT_SESSION_SAVE_SECONDS does? Sometimes when I restart the container it comes back up with a default save location for torrents completed before the restart but after the last RT_SESSION_SAVE_SECONDS.

**Comment by hitboxr** [comment]
I'm no expert but this sounds like a case of docker killing the container before rtorrent finishes gracefully shutting down. Have you tried this?
https://github.com/crazy-max/docker-rtorrent-rutorrent/tree/master?tab=readme-ov-file#increase-docker-timeout-to-allow-rtorrent-to-shutdown-gracefully

**Reply by boomboxnation** [reply]
Oh it is...thing is...I am impatient and actually do want it to die when I say die. I just want to 'save my work' beforehand.

Also the graceful shutdown of rtorrent is a moving target. I would rather just keep my client shutdown for my tracker indicated mean announce time before starting it up again.
